### ISSUE TITLE
unresolved dependency

### ISSUE THREAD

**Issue by ganeshchand** [jamesward/play-auto-refresh#42]
I have `scalaVersion := "2.12.2"` and `sbt.version=0.13.17` and `addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.17")`
I keep getting this error. Is this plugin supported for 2.6.x? Do I need to add a resolver in plugins.sbt?

```
[error] (*:update) sbt.ResolveException: unresolved dependency: com.jamesward#play-auto-refresh;0.0.16: not found
```

**Comment by jamesward** [comment]
It looks like I only published `0.0.16` for sbt `1.0.x` - do you need it for sbt `0.13.x`?

**Comment by mkurz** [comment]
5 years old, no reply, closing
